cookbooks/tutorial/tutorial.prm

```diff
@@ -0,0 +1,122 @@
+include $ASPECT_SOURCE_DIR/cookbooks/convection-box/convection-box.prm
```

Comment:
do you need this at all?

Reply:
Yes. I made a diff and then included only the things I changed. But I can also put this in as a completely separate file.

Reply:
One could argue that this one might be the first one to teach, so it might be worth setting all parameters? Not sure.